problems/27.Check If Two String Arrays are Equivalent.gs

```diff
@@ -0,0 +1,15 @@
+function arrayStringsAreEqual(word1: string[], word2: string[]): boolean {
+    let str1: string = strings(word1)
+    let str2: string = strings(word2)
+    return str1 == str2
```

Comment:
You can optimize here, before you call strings make sure word1 and word2 length is the same, if not they are for sure not the same...